src/guides/drizzle-and-contract-events/index.md

```diff
@@ -78,43 +70,35 @@ truffle(develop)> migrate
 
 Now that we have a test chain running and our smart contracts deployed, let's add a toast notification to the UI.
 
-
-Listen for Contract Events
-==========================
+## Listen for Contract Events
 
 We want to listen for the `SimpleStorage` contract's `StorageSet` event and show a toast notification once it fires.
 
-The front end code is located under the `app` folder. Lets add the notification
-library `react-toastify` to simulate an external interaction.
+The front end code is located under the `app` folder. Lets add the notification library `react-toastify` to simulate an external interaction From the project directory:
 
 ```shell
-$ cd app
-$ npm install react-toastify
+cd app
+npm install react-toastify
 ```
 
 For the sake of simplicity, we will work in one file, `./app/src/middleware/index.js`.
 
 ```shell
-$ mkdir ./src/middleware
-$ touch ./src/middleware/index.js
+mkdir ./src/middleware
+touch ./src/middleware/index.js
 ```
 
-Import `EventActions` and `generateStore` from Drizzle as well as
+In `./app/src/middleware/index.js`, import `EventActions` and `generateStore` from Drizzle as well as
 toast from `react-toastify`, and `drizzleOptions`.
 
 ```javascript
-// ./app/middleware/index.js
+// ./app/src/middleware/index.js
 import { generateStore, EventActions } from '@drizzle/store'
 import drizzleOptions from '../drizzleOptions'
 import { toast } from 'react-toastify'
 ```
 
-The action `EventActions.EVENT_FIRED` is emitted whenever a contract event is
-detected in a Block. We will gain access to it by registering a custom
-middleware with the Redux store. As you know, Redux middleware comprises a set
-of functions executed in a sequence that processes each dispatched actions
-before passing them to Reducers.
-
+The action `EventActions.EVENT_FIRED` is emitted whenever a contract event is detected in a Block. We will gain access to it by registering a custom middleware with the Redux store. As you may know, Redux middleware comprises a set of functions, executed in a sequence, that process each dispatched action before passing it to Reducers. Add the following:
```

Comment:
I feel like Block doesn't need to be capitalized? But if it also capitalized elsewhere please ignore this!

```suggestion
The action `EventActions.EVENT_FIRED` is emitted whenever a contract event is detected in a block. We will gain access to it by registering a custom middleware with the Redux store. As you may know, Redux middleware comprises a set of functions, executed in a sequence, that process each dispatched action before passing it to Reducers. Add the following:
```